src/include/txn.h

```diff
@@ -328,28 +328,29 @@ struct __wt_txn {
 
 /* AUTOMATIC FLAG VALUE GENERATION START */
 #define WT_TXN_AUTOCOMMIT 0x000001u
-#define WT_TXN_ERROR 0x000002u
-#define WT_TXN_HAS_ID 0x000004u
-#define WT_TXN_HAS_SNAPSHOT 0x000008u
-#define WT_TXN_HAS_TS_COMMIT 0x000010u
-#define WT_TXN_HAS_TS_DURABLE 0x000020u
-#define WT_TXN_HAS_TS_PREPARE 0x000040u
-#define WT_TXN_IGNORE_PREPARE 0x000080u
-#define WT_TXN_PREPARE 0x000100u
-#define WT_TXN_READONLY 0x000200u
-#define WT_TXN_RUNNING 0x000400u
-#define WT_TXN_SHARED_TS_DURABLE 0x000800u
-#define WT_TXN_SHARED_TS_READ 0x001000u
-#define WT_TXN_SYNC_SET 0x002000u
-#define WT_TXN_TS_COMMIT_ALWAYS 0x004000u
-#define WT_TXN_TS_COMMIT_KEYS 0x008000u
-#define WT_TXN_TS_COMMIT_NEVER 0x010000u
-#define WT_TXN_TS_DURABLE_ALWAYS 0x020000u
-#define WT_TXN_TS_DURABLE_KEYS 0x040000u
-#define WT_TXN_TS_DURABLE_NEVER 0x080000u
-#define WT_TXN_TS_ROUND_PREPARED 0x100000u
-#define WT_TXN_TS_ROUND_READ 0x200000u
-#define WT_TXN_UPDATE 0x400000u
+#define WT_TXN_BEGINNING 0x000002u /* Set only for explicit transactions. */
```

Comment:
How about `WT_TXN_STARTING`?

Reply:
The function name is `__wt_txn_begin`. To be consistent, I think beginning is better.

Reply:
Yes, agree to keep the flag same as with the function name.